internal/pkg/config/config_test.go

```diff
@@ -78,6 +78,22 @@ func testToMapStr(t *testing.T) {
 	assert.True(t, reflect.DeepEqual(m, nm))
 }
 
+func TestCommaParsing(t *testing.T) {
+	_ = os.Setenv("testname", "motmot")
+	// test to make sure that we don't blow up the parsers when we have a `,` in a string
+	inMap := map[string]interface{}{
+		"test": "startsWith('${testname}','motmot')",
+	}
+	outMap := map[string]interface{}{
+		"test": "startsWith('motmot','motmot')",
+	}
+	c := MustNewConfigFrom(inMap)
+	parsedMap, err := c.ToMapStr()
+	require.NoError(t, err)
+	t.Logf("got :%#v", parsedMap)
+	assert.True(t, reflect.DeepEqual(outMap, parsedMap))
```

Comment:
[Blocker]
Did you use `DeepEqual` because the normal `assert.Equal` won't do the same check?
If so, could you add a message and also the diff? Otherwise if it fails the message will be just something like "false is not true", which isn't helpful at all